### ISSUE TITLE
Add a folder with the all gathered data 

### ISSUE THREAD

**Issue by leonprou** [liquify-validation/Fuse_spark_test_script#2]
The repo should include a folder with all gathered data (from the start to latest block):
- cycles
- results
(not need to include the pieCharts IMO as generated fast)

This folder shouldn't be overwritten when running the script

**Comment by Andrew-Pohl** [comment]
Agreed sounds like a good idea I suggest instead of instantly overwriting the old folder on re run we copy it and delete the old folder at the end on a successful run (learnt this the hard way before!)

**Comment by leonprou** [comment]
@Andrew-Pohl Sharon suggested that we uploaded this to S3/google drive and put a public link to it in the repo and on the forum. So the data can be fetched easily with wget for example.

**Comment by Andrew-Pohl** [comment]
> @Andrew-Pohl Sharon suggested that we uploaded this to S3/google drive and put a public link to it in the repo and on the forum. So the data can be fetched easily with wget for example.

We could wget straight from github too?
